File: 백준/Bronze/11098. 첼시를 도와줘！/첼시를 도와줘！.cc
#include <iostream>
using namespace std;

int main() {
	
	int n;
	cin >> n;

	for (int i = 0; i < n; i++)
	{
		int playerN;
		cin >> playerN;

		int price;
		int maxPrice = 0;
		string name;
		string pick;
		for (int j = 0; j < playerN; j++)
		{
			cin >> price;
			cin.ignore();
			cin >> name;

			if (price >= maxPrice)
			{
				maxPrice = price;
				pick = name;
			}
		}

		cout << pick << endl;
	}
}

File: 백준/Bronze/11098. 첼시를 도와줘！/첼시를 도와줘！.java
import java.util.Scanner;

public class Main {
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);

        int n = sc.nextInt();

        for (int i = 0; i < n; i++) {
            int playerN = sc.nextInt();
            int maxPrice = 0;
            String pick = "";

            for (int j = 0; j < playerN; j++) {
                int price = sc.nextInt();
                String name = sc.next();

                if (price >= maxPrice) {
                    maxPrice = price;
                    pick = name;
                }
            }

            System.out.println(pick);
        }
    }
}